client/src/assets/js/app/typing-lesson.js

```diff
@@ -113,6 +115,22 @@
       isRunning: function() {
         return (this.state === LevelStates.Running);
       },
+      incorrectCharsSorted: function() {
+        let charsCount = this.result.missed_chars_count;
+
+        for(let key in charsCount){
+          this.sortedIncorrectChars.push({
+            letter: key,
+            count: charsCount[key],
+            percent: (charsCount[key] / this.totalWrongCount * 100), 
```

Comment:
We should save `charsCount[key] / this.totalWrongCount` to a variable since the value is used twice.